main.js

```diff
@@ -151,13 +115,171 @@ const inventory = [
     screenQuality: 'Full HD',
     smartTv: false,
     options: {
-      wifi: false,
-      speech: false,
-      hdr: true,
-      bluetooth: false,
-      ambiLight: false,
+        wifi: false, speech: false, hdr: true, bluetooth: false, ambiLight: false,
     },
     originalStock: 10,
     sold: 8,
-  },
-];
\ No newline at end of file
+},];
+
+//opdracht-1//
+
+console.log("-----1A-----")
+
+const tvNames = inventory.map((inventoryType) => {
+    return ({"type": inventoryType.type, "name": inventoryType.name})
+})
+
+console.log(tvNames)
+
+console.log("-----1B-----")
+
+const tvSoldOut = inventory.filter((inventoryEmpty) => {
+    return inventoryEmpty.originalStock === inventoryEmpty.sold
+})
+
+console.log(tvSoldOut)
+
+console.log("-----1C-----")
+
+const tvAmbiLightList = inventory.filter((inventoryAL) => {
+    return inventoryAL.options.ambiLight === true
+})
+
+console.log(tvAmbiLightList)
+
+console.log("-----1D-----")
+
+function sortPriceLoHi(array) {
+    return array.sort((a, b) => a.price - b.price)
+}
+
+sortPriceLoHi(inventory)
+
+console.log(inventory)
+
+//opdracht 2//
+console.log("-----2A-----")
+
+let tvSold = 0
+
+for (let i = 0; i < inventory.length; i++) {
+    tvSold = tvSold + inventory[i].sold
```

Comment:
Dit kun je korter schrijven
```suggestion
    tvSold += inventory[i].sold;
```